src/components/SearchPage/SearchPageComponent.js

```diff
@@ -18,6 +19,11 @@ export function SearchPageComponent() {
     1000,
   );
 
+  const handleClick = note => {
+    const data = { ...note, label: note.name };
+    dispatch(searchResultClickedAction({ note: data, edges: [] }));
```

Comment:
I don't think we should pass `edges` here. The action should just tell what happened: a search result was clicked. It shouldn't say what should happened with mind map edges. That's responsibility of a reducer.